content/operate/kubernetes/reference/supported_k8s_distributions.md

```diff
@@ -12,6 +12,8 @@ weight: 30
 
 Each release of Redis Enterprise for Kubernetes is thoroughly tested against a set of Kubernetes distributions. The table below lists Redis Enterprise for Kubernetes versions and the Kubernetes distributions they support.
 
+{{<note>}}Each platform is tested with the default configuration, including storage, network, security, and container runtime components.”{{</note>}}
```

Comment:
Kinda nitpicky, but I think `its` better emphasizes that the default configuration is unique to each platform, rather than _some_ default configuration that is applied similarly to all platforms.
```suggestion
{{<note>}}Each platform is tested with its default configuration, including storage, network, security, and container runtime components.”{{</note>}}
```